### ISSUE TITLE
Reddit won't work anymore for maybe a month

### ISSUE THREAD

**Issue by mr-wong** [yt-dlp/yt-dlp#9912]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting that yt-dlp is broken on a **supported** site
- [X] I've verified that I have **updated yt-dlp to nightly or master** ([update instructions](https://github.com/yt-dlp/yt-dlp#update-channels))
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

n/a

### Provide a description that is worded well enough to be understood

As the title states. Thanks in advance.

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
[debug] Command-line config: ['-vU', 'https://www.reddit.com/r/JustGuysBeingDudes/comments/1cqfqwm/abra_cadabra/']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version master@2024.05.12.230604 from yt-dlp/yt-dlp-master-builds [85ec2a337] (win_exe)
[debug] Python 3.8.10 (CPython AMD64 64bit) - Windows-10-10.0.22631-SP0 (OpenSSL 1.1.1k  25 Mar 2021)
[debug] exe versions: ffmpeg 6.0-essentials_build-www.gyan.dev (setts), ffprobe 6.0-essentials_build-www.gyan.dev
[debug] Optional libraries: Cryptodome-3.20.0, brotli-1.1.0, certifi-2024.02.02, curl_cffi-0.5.10, mutagen-1.47.0, requests-2.31.0, sqlite3-3.35.5, urllib3-2.2.1, websockets-12.0
[debug] Proxy map: {}
[debug] Request Handlers: urllib, requests, websockets, curl_cffi
[debug] Loaded 1803 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp-master-builds/releases/latest
Latest version: master@2024.05.12.230604 from yt-dlp/yt-dlp-master-builds
yt-dlp is up to date (master@2024.05.12.230604 from yt-dlp/yt-dlp-master-builds)
[Reddit] Extracting URL: https://www.reddit.com/r/JustGuysBeingDudes/comments/1cqfqwm/abra_cadabra/
[Reddit] 1cqfqwm: Downloading JSON metadata
WARNING: [Reddit] 1cqfqwm: Failed to parse JSON: Expecting value in '': line 1 column 1 (char 0)
[Reddit] www.reddit.com request failed, retrying with old.reddit.com
[Reddit] 1cqfqwm: Downloading JSON metadata
ERROR: [Reddit] 1cqfqwm: 1cqfqwm: Failed to parse JSON (caused by JSONDecodeError("Expecting value in '': line 1 column 1 (char 0)")); please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
  File "yt_dlp\extractor\common.py", line 734, in extract
  File "yt_dlp\extractor\reddit.py", line 208, in _real_extract
  File "yt_dlp\extractor\common.py", line 1127, in download_content
  File "yt_dlp\extractor\common.py", line 1094, in download_handle
  File "yt_dlp\extractor\common.py", line 1082, in parse
  File "yt_dlp\extractor\common.py", line 1069, in _parse_json
  File "yt_dlp\extractor\common.py", line 1052, in __print_error

  File "yt_dlp\utils\_utils.py", line 552, in decode
  File "json\decoder.py", line 337, in decode
  File "json\decoder.py", line 355, in raw_decode
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "yt_dlp\extractor\common.py", line 1066, in _parse_json
  File "json\__init__.py", line 370, in loads
  File "yt_dlp\utils\_utils.py", line 560, in decode
json.decoder.JSONDecodeError: Expecting value in '': line 1 column 1 (char 0)
```

**Comment by bashonly** [comment]
Are you sure that you actually did this:

> - [x] I've checked that all provided URLs are playable in a browser with the same IP and same login details

Your IP is being blocked by reddit. Are you using a VPN? Or otherwise using yt-dlp from a data center IP address? Have you made a large volume of requests to reddit recently? 

Most likely a duplicate of #9714

**Comment by mr-wong** [comment]
Ahh, sorry, completely forgot that I am actually logged in. So, the URLs are playable while I am logged in, but they aren't when I am not. Both my real IP and VPN are blocked by Reddit, though. Not sure why, I simply surf like a regular user. Will file a ticket there. Anyway, thank you for your effort. :)
